### PR TITLE
Update User login

### PR DESCRIPTION
**Pull request by AndreSand**
Update User login

### CODE REVIEW

**Comment by dbykovsky** [comment]
That's why you have conflicts. Once again. The job that you are trying to push is already done https://github.com/dbykovsky/sumofusM/commit/3a414064c7218b132d596009828c609192e91816
@alberto-campos

**Comment by AndreSand** [comment]
Added zipcode to registration, prepopulated zip when signing petition...

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -72,11 +72,13 @@
   <string name="title_activity_signup">Sign up</string>
   <string name="logout">Log Out</string>
   <string name="prompt_username">User name</string>
+  <string name="prompt_zipcode">ZipCode</string>
   <string name="prompt_useremail">User e-mail</string>
   <string name="prompt_password">Password</string>
   <string name="prompt_password_again">Password Again</string>
   <string name="error_intro">"Please "</string>
   <string name="error_blank_username">enter a name</string>
+  <string name="error_blank_zipcode">enter a username</string>
```
Comment (dbykovsky) [inline app/src/main/res/values/strings.xml:81]: This should be a "enter a zip code" I believe.

**Comment by dbykovsky** [comment]
Do we need to require zip code upon the registration? We never discussed that before @alberto-campos

**Comment by alberto-campos** [comment]
Sum of Us requires 2 fields to sign a petition.
- E-mail address is required.
- Postal/ZIP Code is required.

Name / Username is optional.

For a donation all fields are required (necessary to complete a donation).

**Comment by dbykovsky** [comment]
Those fields are present on sign in petition page, but we don't need to require zip on sign u screen. Right?

**Comment by AndreSand** [comment]
@dbykovsky @alberto-campos everything ok to merge?

**Comment by dbykovsky** [comment]
We are in the code freeze stage. have you received an email last night.
Moreover, this functionality is not critical and was never discussed
withing the team. Alberto and me commented on this pull request last night.
We don't require zip code upon sign in. Please, review comment on this PR
form yesterday.

**Comment by AndreSand** [comment]
Ok I closed the pull.

**Pull request closed without merging**